core/src/display_object/movie_clip.rs

```diff
@@ -4611,7 +4604,7 @@ impl<'gc, 'a> MovieClip<'gc> {
                         }
                     }
                     Err(e) => tracing::error!(
-                        "Got AVM2 error {e:?} when attempting to assign symbol class {name:?}",
```

Comment:
Why not print the error anymore?

Reply:
We still print the error, but not the name of the symbolclass, since that should already appear in the stack trace